From 62b70f8299e67a879097041b92faa489e0fa89a4 Mon Sep 17 00:00:00 2001
From: "Christopher N. Hesse" <raymanfx@gmail.com>
Date: Mon, 19 Feb 2018 23:20:14 +0100
Subject: [PATCH 09/35] power: Allow POWER_HINT_DISABLE_TOUCH in low power mode

Remove the unused len var while we're at it.

Change-Id: Iabaf3b8704072a8758b213a54530eaffbf0c7ae5
---
 power/power.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/power/power.c b/power/power.c
index ac61d3f..a34a5b5 100644
--- a/power/power.c
+++ b/power/power.c
@@ -477,11 +477,10 @@ static void samsung_power_hint(struct power_module *module,
                                   void *data)
 {
     struct samsung_power_module *samsung_pwr = (struct samsung_power_module *) module;
-    int len;
 
     /* Bail out if low-power mode is active */
     if (current_power_profile == PROFILE_POWER_SAVE && hint != POWER_HINT_LOW_POWER
-            && hint != POWER_HINT_SET_PROFILE) {
+            && hint != POWER_HINT_SET_PROFILE && hint != POWER_HINT_DISABLE_TOUCH) {
         ALOGV("%s: PROFILE_POWER_SAVE active, ignoring hint %d", __func__, hint);
         return;
     }
-- 
2.17.1

